compiler/testData/codegen/boxInline/delegatedProperty/local.kt

```diff
@@ -1,4 +1,3 @@
-// IGNORE_BACKEND: JVM_IR
```

Comment:
It's not clear why this test is affected, should it be merged with one of previous commits?

Reply:
It's affected because the delegated property's accessors are extracted by LocalDeclarationsLowering into private methods, which with this commit are called through public accessors as an accidental workaround. The lambda methods should probably be inlined or at least have the visibility of the function itself.